### ISSUE TITLE
Management Command "rendervariations" throws an AttributeError with Current Version of progressbar2 (3.3.0)

### ISSUE THREAD

**Issue by bltravis** [codingjoe/django-stdimage#56]
Looks like they did a major re-factor of the progressbar2 app, and the class referenced no longer exists. See: https://github.com/WoLpH/python-progressbar/blob/develop/progressbar/widgets.py
```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/django/domains/mydomain.com/lib/python3.4/site-packages/django/core/management/__init__.py", line 351, in execute_from_command_line
    utility.execute()
  File "/home/django/domains/mydomain.com/lib/python3.4/site-packages/django/core/management/__init__.py", line 343, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/django/domains/mydomain.com/lib/python3.4/site-packages/django/core/management/__init__.py", line 190, in fetch_command
    klass = load_command_class(app_name, subcommand)
  File "/home/django/domains/mydomain.com/lib/python3.4/site-packages/django/core/management/__init__.py", line 40, in load_command_class
    module = import_module('%s.management.commands.%s' % (app_name, name))
  File "/usr/local/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/home/django/domains/mydomain.com/lib/python3.4/site-packages/stdimage/management/commands/rendervariations.py", line 19, in <module>
    class MemoryUsageWidget(progressbar.widgets.Widget):
AttributeError: 'module' object has no attribute 'Widget'
```

**Comment by codingjoe** [comment]
@bltravis thanks, seems like it.
You should freeze the dependency for now. I don't know how fast I can fix it tho. Don't know if I release an intermediate release that freezes progressbar2.

**Comment by bltravis** [comment]
I'll see if I can't come up with a patch this weekend. Happen to know a working version # for progressbar2, so I can see what the expected behavior is?

**Comment by codingjoe** [comment]
2.7.4 should do the trick.

**Comment by codingjoe** [comment]
@bltravis I froze the dependency for now. I will look into upgrading asap.

**Comment by codingjoe** [comment]
Released in 2.1.1
